README.md

```diff
@@ -121,6 +121,11 @@ have already created it in the target rootfs.
 command execution through `TIOCSTI` commands
 (see [CVE-2017-5226](https://github.com/containers/bubblewrap/issues/142)).
 
+On a practical note, in a shell that supports
+[brace expension](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Brace-Expansion)
+(e.g. Bash), the duplication in arguments like `--ro-bind /usr /usr` can be
+resolved as `--ro-bind /usr{,}` or `--ro-bind {,}/usr`.
```

Comment:
I don't think a shell scripting tip is important enough to justify the space it takes up in the README: everything extraneous we put here is distracting the reader from more important factors, like "it's up to the author of the bubblewrap command-line to make it a security boundary".

Reply:
@smcv fair enough, as I said, I can drop it. Will do in a minute.

Reply:
Dropped now